origin.spec

```diff
@@ -46,16 +46,16 @@
 %endif
 %{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
 
-%if "%{dist}" == ".el7aos"
+%if 0%{?epel}
 %global package_name openshift
 %global product_name OpenShift
 %else
-%global package_name openshift
-%global product_name OpenShift
+%global package_name okd
+%global product_name OKD
 %endif
 
-%{!?version: %global version 0.0.1}
-%{!?release: %global release 1}
+%{!?version: %global version 4.1.0}
```

Comment:
I don’t think we should be tying this version to a top level version, the current plan is that component version numbers are completely component dependent and likely not to be managed under the covers (the rpm may vary wildly over time).  Probably better to start at 4.0.0

Reply:
This was also taken from the auto-generation. Set to 4.0.0 now.